### PR TITLE
edit TestDataStructures.c [@Noura,@ABadr]

### PR DESCRIPTION
**Pull request by nouragmal**

### CODE REVIEW

File: TestDataStructures.c [hunk 1]
```diff
@@ -82,12 +82,24 @@ TEST(LinkedQueue_byLinkedList)
...
 	 EXPECT_EQ(60,top);
+
 	 LinkedQueue_DeQueue(&queue);
 	 LinkedQueue_DeQueue(&queue);
 	 size=LinkedQueue_GetSize(&queue);
 	 EXPECT_EQ(4,size);
+
+	 LinkedQueue_ClearQueue(&queue);
+	 uint8_t x=LinkedQueue_IsEmpty(&queue);
+	 EXPECT_EQ(1,x);
```
Comment (AbdelrahmanBadr22) [inline TestDataStructures.c:97]: There are method called `EXPECT_TRUE` and `EXPECT_FALSE` that used with function that return `true` or `false`

File: TestDataStructures.c [hunk 2]
```diff
@@ -82,12 +82,24 @@ TEST(LinkedQueue_byLinkedList)
...
+	 
+	 LinkedQueue_Init(&queue);
+	 LinkedQueue_EnQueue(&queue,5);
+	 LinkedQueue_EnQueue(&queue,7);
+	 uint8_t y = LinkedQueue_IsFull(&queue);
+	 EXPECT_EQ(0,y);
```
Comment (AbdelrahmanBadr22) [inline TestDataStructures.c:103]: There are method called `EXPECT_TRUE` and `EXPECT_FALSE` that used with function that return `true` or `false`

File: TestDataStructures.c [hunk 3]
```diff
@@ -82,12 +82,24 @@ TEST(LinkedQueue_byLinkedList)
...
 	 EXPECT_EQ(4,size);
+
+	 LinkedQueue_ClearQueue(&queue);
+	 uint8_t x=LinkedQueue_IsEmpty(&queue);
+	 EXPECT_EQ(1,x);
+	 
+	 LinkedQueue_Init(&queue);
+	 LinkedQueue_EnQueue(&queue,5);
+	 LinkedQueue_EnQueue(&queue,7);
+	 uint8_t y = LinkedQueue_IsFull(&queue);
```
Comment (AbdelrahmanBadr22) [inline TestDataStructures.c:102]: make the variable has a clear name 
هنرجع كتاب clean code ف السيشن بعدكدا يا نورا يا جمال

File: TestDataStructures.c [hunk 4]
```diff
@@ -82,12 +82,24 @@ TEST(LinkedQueue_byLinkedList)
...
 	 LinkedQueue_GetTop(&queue,&top);
 	 EXPECT_EQ(60,top);
+
 	 LinkedQueue_DeQueue(&queue);
 	 LinkedQueue_DeQueue(&queue);
 	 size=LinkedQueue_GetSize(&queue);
 	 EXPECT_EQ(4,size);
+
+	 LinkedQueue_ClearQueue(&queue);
+	 uint8_t x=LinkedQueue_IsEmpty(&queue);
```
Comment (AbdelrahmanBadr22) [inline TestDataStructures.c:96]: make the variable has a clear name 
هنرجع كتاب clean code ف السيشن بعدكدا يا نورا يا جمال

**Review by AbdelrahmanBadr22 — CHANGES_REQUESTED** [review]
@nouragmal 
Your code quality is indisputable but changing the highlighted lines will make it better

File: TestDataStructures.c (hunk 4 above)
Comment (nouragmal) [reply TestDataStructures.c:96]: جت على نورا جمال 🌚

**Pull request closed without merging**